templates/ansible/cloudformation.j2

```diff
@@ -0,0 +1,5 @@
+#jinja2:lstrip_blocks:True,trim_blocks:True
```

Comment:
What is the purpose of this cloudformation jinja template?

Reply:
The cloudformation jinja2 template exists because ansible cannot parse variable names inside variable names, it only does one pass to resolve variable names so 

```
{{ elbaccountid.{{ aws.region}} }}
```

does not work, it will resolve the key so it constructs the variable name "elbaccountid.ap-southeast-2" but it will not return the value that it contains as ansible will only resolve variables once.

The Jinja2 template exists so that the value can be exported and reimported. Im open to suggestions on how to clean it up as it does seem inelegant.

Reply:
Have you tried {{ elbaccountid[aws.region] }} ?

Reply:
returns undefined variable.

Reply:
I meant use the replace that you supplied on the separate jinja template... `{{ elbaccountid[aws.region | replace('-', '_') }}` . This is just a pseudocode, you already have the exact syntax on the separate jinja template, my point is that have you tried it on the ansible playbook itself ?

Reply:
Can confirm it works and returned expected output. Cheers for that